### ISSUE TITLE
Android WebDriverException: Message: A new session could not be created

### ISSUE THREAD

**Issue by qaDream77** [appium/appium#6694]
## The problem

`WebDriverException: Message: A new session could not be created. (Original error: com.business.project/.SplashActivity never started. Current: com.business.project/.MainActivity)`

## Environment

* Appium version 1.4.16
* Ubuntu 14.04
* Node.js version 0.12.15
* Android on Genymotion

## Details

I'm watching it open the application and get to the main page of my application, then it throws that error. Can anyone provide some insight please? I didn't have this issue before but i also don't remember what the latest stable version of appium for me was. Upgrading to 1.5 is not an option, i don't have to update all my scripts since deprecated methods were removed. 

##Thank you!

## Appium.log
```

[36minfo[39m: Welcome to Appium v1.4.16 (REV ae6877eff263066b26328d457bd285c0cc62430d)
[36minfo[39m: Appium REST http interface listener started on 0.0.0.0:4723
[36minfo[39m: Console LogLevel: debug
[36minfo[39m: [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m [90m{"desiredCapabilities":{"deviceName":"Custom Phone - 4.4.4 - API 19","app":"/home/user/Documents/My Projects/Mobile/Scripts/resource/../Android/app/MyMobile.apk","appWaitPackage":"com.business.project","platformVersion":"6.0","platformName":"Android","automationName":"appium"}}[39m
[36minfo[39m: Client User-Agent string: Python-urllib/2.7
[36minfo[39m: [debug] No appActivity desired capability or server param. Parsing from apk.
[36minfo[39m: [debug] No appPackage desired capability or server param. Parsing from apk.
[36minfo[39m: [debug] Using local app from desired caps: /home/user/Documents/My Projects/Mobile/Scripts/Android/app/MyMobile.apk
[36minfo[39m: [debug] Creating new appium session 20b08ec4-37b3-4c8f-b0e8-bbcd8d0d78ec
[36minfo[39m: Starting android appium
[36minfo[39m: [debug] Getting Java version
[36minfo[39m: Java version is: 1.7.0_101
[36minfo[39m: [debug] Checking whether adb is present
[36minfo[39m: [debug] Using adb from /home/user/.android/android-sdk-linux/platform-tools/adb
[36minfo[39m: [debug] Parsing package and activity from app manifest
[36minfo[39m: [debug] Checking whether aapt is present
[36minfo[39m: [debug] Using aapt from /home/user/.android/android-sdk-linux/build-tools/23.0.1/aapt
[36minfo[39m: [debug] Extracting package and launch activity from manifest.
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/build-tools/23.0.1/aapt dump badging "/home/user/Documents/My Projects/Mobile/Scripts/Android/app/MyMobile.apk"
[36minfo[39m: [debug] badging package: com.business.project
[36minfo[39m: [debug] badging act: com.business.project.SplashActivity
[36minfo[39m: [debug] Parsed package and activity are: com.business.project/com.business.project.SplashActivity
[36minfo[39m: [debug] Using fast reset? true
[36minfo[39m: [debug] Preparing device for session
[36minfo[39m: [debug] Checking whether app is actually present
[36minfo[39m: Retrieving device
[36minfo[39m: [debug] Trying to find a connected android device
[36minfo[39m: [debug] Getting connected devices...
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb devices
[36minfo[39m: [debug] 1 device(s) connected
[36minfo[39m: Found device 192.168.56.102:5555
[36minfo[39m: [debug] Setting device id to 192.168.56.102:5555
[36minfo[39m: [debug] Waiting for device to be ready and to respond to shell commands (timeout = 5)
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 wait-for-device
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "echo 'ready'"
[36minfo[39m: [debug] Starting logcat capture
[36minfo[39m: [debug] Getting device API level
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "getprop ro.build.version.sdk"
[36minfo[39m: [debug] Device is at API Level 19
[36minfo[39m: Device API level is: 19
[36minfo[39m: [debug] Extracting strings for language: default
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "getprop persist.sys.language"
[36minfo[39m: [debug] Current device persist.sys.language: en
[36minfo[39m: [debug] java -jar "/home/user/.linuxbrew/lib/node_modules/appium/node_modules/appium-adb/jars/appium_apk_tools.jar" "stringsFromApk" "/home/user/Documents/My Projects/Mobile/Scripts/Android/app/MyMobile.apk" "/tmp/com.business.project" en
[36minfo[39m: [debug] No strings.xml for language 'en', getting default strings.xml
[36minfo[39m: [debug] java -jar "/home/user/.linuxbrew/lib/node_modules/appium/node_modules/appium-adb/jars/appium_apk_tools.jar" "stringsFromApk" "/home/user/Documents/My Projects/Mobile/Scripts/Android/app/MyMobile.apk" "/tmp/com.business.project"
[36minfo[39m: [debug] Reading strings from converted strings.json
[36minfo[39m: [debug] Setting language to default
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 push "/tmp/com.business.project/strings.json" /data/local/tmp
[36minfo[39m: [debug] Checking whether aapt is present
[36minfo[39m: [debug] Using aapt from /home/user/.android/android-sdk-linux/build-tools/23.0.1/aapt
[36minfo[39m: [debug] Retrieving process from manifest.
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/build-tools/23.0.1/aapt dump xmltree "/home/user/Documents/My Projects/Mobile/Scripts/Android/app/MyMobile.apk" AndroidManifest.xml
[36minfo[39m: [debug] Set app process to: com.business.project
[36minfo[39m: [debug] Not uninstalling app since server not started with --full-reset
[36minfo[39m: [debug] Checking app cert for /home/user/Documents/My Projects/Mobile/Scripts/Android/app/MyMobile.apk.
[36minfo[39m: [debug] executing cmd: java -jar /home/user/.linuxbrew/lib/node_modules/appium/node_modules/appium-adb/jars/verify.jar "/home/user/Documents/My Projects/Mobile/Scripts/Android/app/MyMobile.apk"
[36minfo[39m: [debug] App already signed.
[36minfo[39m: [debug] Zip-aligning /home/user/Documents/My Projects/Mobile/Scripts/Android/app/MyMobile.apk
[36minfo[39m: [debug] Checking whether zipalign is present
[36minfo[39m: [debug] Using zipalign from /home/user/.android/android-sdk-linux/build-tools/23.0.1/zipalign
[36minfo[39m: [debug] Zip-aligning apk.
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/build-tools/23.0.1/zipalign -f 4 "/home/user/Documents/My Projects/Mobile/Scripts/Android/app/MyMobile.apk" /tmp/116625-30362-524930/appium.tmp
[36minfo[39m: [debug] MD5 for app is 3607175f0edd5e8940dc87157ea59961
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "ls /data/local/tmp/3607175f0edd5e8940dc87157ea59961.apk"
[36minfo[39m: [debug] Getting install status for com.business.project
[36minfo[39m: [debug] Getting device API level
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "getprop ro.build.version.sdk"
[36minfo[39m: [debug] Device is at API Level 19
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "pm list packages -3 com.business.project"
[36minfo[39m: [debug] App is installed
[36minfo[39m: App is already installed, resetting app
[36minfo[39m: [debug] Running fast reset (stop and clear)
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "am force-stop com.business.project"
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "pm clear com.business.project"
[36minfo[39m: [debug] Forwarding system:4724 to device:4724
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 forward tcp:4724 tcp:4724
[36minfo[39m: [debug] Pushing appium bootstrap to device...
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 push "/home/user/.linuxbrew/lib/node_modules/appium/build/android_bootstrap/AppiumBootstrap.jar" /data/local/tmp/
[36minfo[39m: [debug] Pushing settings apk to device...
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 install "/home/user/.linuxbrew/lib/node_modules/appium/build/settings_apk/settings_apk-debug.apk"
[36minfo[39m: [debug] Pushing unlock helper app to device...
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 install "/home/user/.linuxbrew/lib/node_modules/appium/build/unlock_apk/unlock_apk-debug.apk"
[36minfo[39m: Starting App
[36minfo[39m: [debug] Attempting to kill all 'uiautomator' processes
[36minfo[39m: [debug] Getting all processes with 'uiautomator'
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "ps 'uiautomator'"
[36minfo[39m: [debug] No matching processes found
[36minfo[39m: [debug] Running bootstrap
[36minfo[39m: [debug] spawning: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell uiautomator runtest AppiumBootstrap.jar -c io.appium.android.bootstrap.Bootstrap -e pkg com.business.project -e disableAndroidWatchers false
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] INSTRUMENTATION_STATUS: numtests=1[39m
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] INSTRUMENTATION_STATUS: stream=[39m
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] io.appium.android.bootstrap.Bootstrap:[39m
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] INSTRUMENTATION_STATUS: id=UiAutomatorTestRunner[39m
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] INSTRUMENTATION_STATUS: test=testRunServer[39m
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] INSTRUMENTATION_STATUS: class=io.appium.android.bootstrap.Bootstrap[39m
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] INSTRUMENTATION_STATUS: current=1[39m
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] INSTRUMENTATION_STATUS_CODE: 1[39m
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Socket opened on port 4724
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Appium Socket Server Ready
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Loading json...
[36minfo[39m: [debug] Waking up device if it's not alive
[36minfo[39m: [debug] Pushing command to appium work queue: ["wake",{}]
[36minfo[39m: [debug] [BOOTSTRAP] [debug] json loading complete.
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Registered crash watchers.
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Client connected
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Got data from client: {"cmd":"action","action":"wake","params":{}}
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Got command of type ACTION
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Got command action: wake
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Returning result: {"value":true,"status":0}
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window"
[36minfo[39m: [debug] Screen already unlocked, continuing.
[36minfo[39m: [debug] Pushing command to appium work queue: ["getDataDir",{}]
[36minfo[39m: [debug] dataDir set to: /data
[36minfo[39m: [debug] Pushing command to appium work queue: ["compressedLayoutHierarchy",{"compressLayout":false}]
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Got data from client: {"cmd":"action","action":"getDataDir","params":{}}
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Got command of type ACTION
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Got command action: getDataDir
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Returning result: {"value":"\/data","status":0}
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Got data from client: {"cmd":"action","action":"compressedLayoutHierarchy","params":{"compressLayout":false}}
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Got command of type ACTION
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Got command action: compressedLayoutHierarchy
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Returning result: {"value":false,"status":0}
[36minfo[39m: [debug] Getting device API level
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "getprop ro.build.version.sdk"
[36minfo[39m: [debug] Device is at API Level 19
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "am start -S -a android.intent.action.MAIN -c android.intent.category.LAUNCHER -f 0x10200000 -n com.business.project/com.business.project.SplashActivity"
[36minfo[39m: [debug] Waiting for pkg "com.business.project" and activity "com.business.project.SplashActivity" to be focused
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Getting focused package and activity
[36minfo[39m: [debug] executing cmd: /home/user/.android/android-sdk-linux/platform-tools/adb -s 192.168.56.102:5555 shell "dumpsys window windows"
[36minfo[39m: [debug] Stopping logcat capture
[36minfo[39m: [debug] Logcat terminated with code null, signal SIGTERM
[36minfo[39m: [debug] Sent shutdown command, waiting for UiAutomator to stop...
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Got data from client: {"cmd":"shutdown"}
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Got command of type SHUTDOWN
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Returning result: {"value":"OK, shutting down","status":0}
[36minfo[39m: [debug] [BOOTSTRAP] [debug] Closed client connection
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] INSTRUMENTATION_STATUS: numtests=1[39m
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] INSTRUMENTATION_STATUS: stream=.[39m
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] INSTRUMENTATION_STATUS: id=UiAutomatorTestRunner[39m
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] INSTRUMENTATION_STATUS: test=testRunServer[39m
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] INSTRUMENTATION_STATUS: class=io.appium.android.bootstrap.Bootstrap[39m
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] INSTRUMENTATION_STATUS: current=1[39m
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] INSTRUMENTATION_STATUS_CODE: 0[39m
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] INSTRUMENTATION_STATUS: stream=[39m
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] Test results for WatcherResultPrinter=.[39m
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] Time: 20.892[39m
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] OK (1 test)[39m
[36minfo[39m: [debug] [90m[UIAUTOMATOR STDOUT] INSTRUMENTATION_STATUS_CODE: -1[39m
[36minfo[39m: [debug] UiAutomator shut down normally
[36minfo[39m: [debug] Cleaning up android objects
[36minfo[39m: [debug] Cleaning up appium session
[36minfo[39m: [debug] Error: com.business.project/.SplashActivity never started. Current: com.business.project/.MainActivity
    at [object Object].<anonymous> (/home/user/.linuxbrew/lib/node_modules/appium/node_modules/appium-adb/lib/adb.js:1314:12)
    at [object Object].<anonymous> (/home/user/.linuxbrew/lib/node_modules/appium/node_modules/appium-adb/lib/adb.js:1258:7)
    at [object Object].<anonymous> (/home/user/.linuxbrew/lib/node_modules/appium/node_modules/appium-adb/lib/adb.js:180:9)
    at ChildProcess.exithandler (child_process.js:745:7)
    at ChildProcess.emit (events.js:110:17)
    at maybeClose (child_process.js:1019:16)
    at Process.ChildProcess._handle.onexit (child_process.js:1091:5)
[36minfo[39m: [debug] Responding to client with error: {"status":33,"value":{"message":"A new session could not be created. (Original error: com.business.project/.SplashActivity never started. Current: com.business.project/.MainActivity)","origValue":"com.business.project/.SplashActivity never started. Current: com.business.project/.MainActivity"},"sessionId":null}
[36minfo[39m: [37m<-- POST /wd/hub/session [39m[31m500[39m[90m 53869.933 ms - 328[39m [90m[39m
```

**Comment by mykola-mokhnach** [comment]
Set the _appActivity_ capability to _com.business.project/.MainActivity_ and _appWaitActivity_ to
_com.business.project/.MainActivity,com.business.project/.SplashActivity_

**Comment by qaDream77** [comment]
@mykola-mokhnach Thank you!

These are my current capabilities

```
platformName=Android
platformVersion=4.4.4
deviceName=192.168.56.101:5555
app=MyMobile.apk
automationName=appium
appWaitPackage=com.business.project
```

and changed it to:


```
platformName=Android
platformVersion=4.4.4
deviceName=192.168.56.101:5555
app=MyMobile.apk
automationName=appium
appWaitPackage=com.business.project
appActivity=com.business.project/.MainActivity
appWaitActivity=com.business.project/.MainActivity,com.business.project/.SplashActivity
```

I'll update once the test is complete.

**Comment by mykola-mokhnach** [comment]
_deviceName_ capability seems to be wrong. It should be either _null_ (for autoselection) or the actual device ID

**Comment by qaDream77** [comment]
@mykola-mokhnach That is correct actually for auto select. I am using Robot Framework Library, unsure if appium bug or library bug but it is a required field and providing the device name appium throws the error in the log saying that it cannot find a device with such a name.

So far it's working! I have a script running every 3 minutes and i'll see how frequently it crashes with that issue.

**Comment by qaDream77** [comment]
@mykola-mokhnach I've been running it stably for the last hour or so every 10 minutes. Looks good. I'll close for now and if it comes back i'll open a new ticket. Thank you!

**Comment by VishJP** [comment]
How to disable appium log on console?

�[36minfo�[39m: Welcome to Appium v1.4.16 (REV ae6877eff263066b26328d457bd285c0cc62430d)
�[36minfo�[39m: Appium REST http interface listener started on 0.0.0.0:4723
�[36minfo�[39m: Console LogLevel: debug
